### ISSUE TITLE
Division by zero error when checking if polygon contains a point

### ISSUE THREAD

**Issue by maxf** [alexbol99/flatten-js#18]
The following code causes a Zero division exception. Because Chrome is struggling with the source map and doesn't point to the right lines as I try to detect where this is happening, I'm lazily submitting it here:
```
             const Flatten = window.flatten;
             const points = [
                 new Flatten.Point(-0.0774582, 51.4791865),
                 new Flatten.Point(-0.0784252, 51.4792941),
                 new Flatten.Point(-0.0774582, 51.4791865)
             ]
             const poly = new Flatten.Polygon();
             poly.addFace(points);
             const pp = new Flatten.Point(-0.07776044568759738, 51.47918678917519);
             console.log(poly.contains(pp));
```

EDIT: reduced number of points causing error

**Comment by alexbol99** [comment]
Bug fixed in v0.6.7
Thank you @maxf  for your cooperation

**Comment by maxf** [comment]
Thanks!
